src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -838,11 +838,22 @@ private function registerAnnotationsConfiguration(array $config, ContainerBuilde
 
     private function registerPropertyAccessConfiguration(array $config, ContainerBuilder $container)
     {
-        $container
+        $definition = $container
             ->getDefinition('property_accessor')
             ->replaceArgument(0, $config['magic_call'])
             ->replaceArgument(1, $config['throw_exception_on_invalid_index'])
         ;
+
+        if (isset($config['cache']) && $config['cache']) {
+            $container->setParameter(
+                'property_accessor.cache.prefix',
```

Comment:
It can also be used if you register your own Doctrine Cache service (relying on this parameter). It allows to register another backend (such as `ArrayCache`, is we drop the in memory array) without having to write an extension userland (this definition can just use this parameter).

And i's to be coherent with validator and serializer (they work the same way).